ckanext/qgov/common/templates/user/login.html

```diff
@@ -0,0 +1,9 @@
+{% ckan_extends %}
+
+{% block sso_form %}
+    <p><a href="{{ h.url_for('user.login') }}/oidc-pkce" class="btn btn-primary"><i class="fa fa-sign-in"></i> Sign in with your Digital ID</a></p>
+{% endblock %}
+
+{% block sso_help_register %}
+    Learn more about using a <a href="https://www.qld.gov.au/qdifaq">Digital ID</a>
```

Comment:
I think its still good to keep the Register button

```
<a href="/user/login/oidc-pkce" class="btn btn-secondary">Register via your Digital ID</a></p>
<p>Learn more about using a <a href="https://www.qld.gov.au/qdifaq">Digital ID</a></p>
```

need to close out the p tag we are in and open it again to get nice spacing.

Reply:
It's actually the same link as login, though...